src/Geno/C++/GUI/Modals/NewItemModal.cpp

```diff
@@ -38,7 +38,7 @@ void NewItemModal::RequestPath( std::string Title, std::filesystem::path Default
 	{
 		m_Title       = std::move( Title );
 		m_Location    = std::move( DefaultLocation );
-		m_Callback    = Callback;
+		m_Callback    = (void*) Callback;
```

Comment:
please use static_cast (or reinterpret_cast if not possible)

Reply:
That freaks out with function pointer conversion - type punning is bad in C++ and static conversion between `void*` and function pointers is disallowed

Reply:
Even with reinterpret_cast?